src/devices/Ssd1351/tests/Ssd1351Tests.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace Iot.Device.Ssd1351.Tests
+{
+    public class Ssd1351Tests
+    {
+        [Fact]
+        public void Test1()
+        {
+            // NOTE: THIS TEST PROJECT IS NOT REQUIRED AND CAN BE DELETED IF NOT USED.
```

Comment:
please delete the test project if not used

Reply:
Done in the latest PR